src/spatial/collision.js

```diff
@@ -353,7 +346,7 @@ Crafty.c("Collision", {
     /**@
```

Comment:
The coments above was assosiated with `_createCollisionHandler` method, I don't know why github renders them here.